i915.c

```diff
@@ -101,6 +101,7 @@ static int i915_add_combinations(struct driver *drv)
 
 	/* Android CTS tests require this. */
 	drv_add_combination(drv, DRM_FORMAT_BGR888, &metadata, BO_USE_SW_MASK);
+	drv_add_combination(drv, DRM_FORMAT_ABGR2101010, &metadata, BO_USE_SW_MASK);
```

Comment:
We can try to use "drv_modify_combination" to help add additional usage to one format, :) 
And if possible, could we add one flag to identify the Internal changes? For example:
#ifdef USE_GRALLOC1
drv_modify_combination
#endif